ServiceBoxWeb/src/main/resources/templates/employee/start.html

```diff
@@ -26,7 +26,7 @@ <h1>Panel pracownika</h1>
             <p class="mt-1 mt-lg-4">
 <!--                <a th:href="@{/employee/clients}" class="btn btn-outline-secondary btn-lg me-2">Klienci</a>-->
```

Comment:
27, 28 Comments should be deleted?

Reply:
Faktycznie, nieładnie zostawiać zakomentowany kod "na później"!